src/printing.py

```diff
@@ -12,6 +12,11 @@
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
 
+print("x is %d, y is %f, z is %s" % (x, round(y, 2), z))
+
 # Use the 'format' string method to print the same thing
+print("x is {}, y is {}, z is {}".format(x, round(y, 2), z))
+
+# Finally, print the same thing using an f-string
 
-# Finally, print the same thing using an f-string
\ No newline at end of file
+print(F"x is {x}, y is {round(y, 2)}, z is {z}")
```

Comment:
Same as above, check out this link for how to round with f-strings https://stackoverflow.com/questions/45310254/fixed-digits-after-decimal-with-f-strings